src/main/java/duke/Messages.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+public class Messages {
+    private static String DOTTED_LINE = "___________________________________________________\n";
+
+    public static void welcomeMessage() {
+        printDivider();
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println("What can I do for you?");
+        printDivider();
+    }
```

Comment:
Consider using verbs for method names to describe the action being executed by the method. Perhaps "welcomeMessage" could be renamed to "printWelcomeMessage"?